wearipedia/devices/fitbit/fitbit_charge_4_gen.py

```diff
@@ -179,6 +180,8 @@ def create_syn_data(start_date, end_date):
     :rtype: defaultdict
```

Comment:
The comment says that the function returns heart_rate data, but it doesn't look like heart rate is being generated or returned here. Let me know if I glossed over something.